village/src/components/Smurfs.js

```diff
@@ -1,14 +1,32 @@
 import React, { Component } from 'react';
-
+import axios from 'axios';
 import Smurf from './Smurf';
 
 class Smurfs extends Component {
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
+  state={
+    smurfs: ["test"],
+  }
+
+componentDidMount() {
+  axios.get('http://localhost:3333/smurfs')
+  .then(response => {
+      console.log(response.data);
+      this.setState({ smurfs: response.data });
+    })
+  .catch(error => {
+      console.log(`There was an error getting smurfs: ${error}`);
+    });
+}
+
 
   render() {
+    console.log(this.state.smurfs);
+    console.log(this.state);
     return (
       <div className="Smurfs">
         <h1>Smurf Village</h1>
+        <button onClick={(event) => this.componentDidMount()}> Get Smurfs </button>
```

Comment:
Haha. This is kinda hacky but I understand why you had to do it. To be honest, I would've refactored and kept all state in the parent component (`App.js`) that way when you submit a new Smurf, it will re-render and show the changes.